src/main/java/com/hobbycollection/HobbyCollectionRESTController.java

```diff
@@ -10,14 +12,19 @@
     @Autowired
     private ICollectionService collectionService;
 
+    Logger log = LoggerFactory.getLogger(this.getClass());
+
     /**
      * Fetches a collection by the id
      * @param id
      * @return URL
      */
     @GetMapping("/api/Collection/fetchByid")
     public Collection collectionFetchById(@RequestParam(value="id") int id){
-        return new Collection("test name", "http://some.url","test description");
+        return new Collection(
+                "test name",
+                "http://some.url",
+                "test description");
```

Comment:
I fixed the formatting of the code.
Source:
https://www.baeldung.com/java-clean-code